woocommerce/woocommerce-gateway-stripe
# Add list of payment methods that are not included for merchant

Fixes #2173

## Changes proposed in this Pull Request:








This PR adds a list of payment methods that are not included for the merchant. It helps the merchant identify what other payment methods are not included for their store.

![image](https://user-images.githubusercontent.com/6844516/144281904-4e2f03e0-be59-453c-add6-9877f2d83b55.png)

## Testing instructions

1. Ensure the plugin ([woocommerce-gateway-stripe](https://github.com/woocommerce/woocommerce-gateway-stripe)) is [installed and configured according to the docs](https://woocommerce.com/document/stripe/#setup-and-configuration).
2. Go to `wp-admin > WooCommerce > Settings > Payments > Stripe`.
3. In the `Payment methods` section (see image above), verify that the unavailable payment methods' icons appear at the bottom, next to the `Get more payment methods` button. Also note that if more than 3 payment methods are not available, a `+ # more` text element appears.





---


**Post merge**
